src/components/form-elements/_select.scss

```diff
@@ -75,10 +75,6 @@ $includeHtml: false !default;
 
     &--invalid {
       border: 2px solid $selectInvalidBorderColor;
-
```

Comment:
This was breaking select
<img width="235" alt="screen shot 2015-10-05 at 09 41 30" src="https://cloud.githubusercontent.com/assets/316313/10275258/56a7c4fe-6b46-11e5-911a-6716160b33cb.png">